tests/slice_by_condition_test.py

```diff
@@ -1,10 +1,40 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+# @file
 # @author Simon Heybrock
 import pytest
 import scipp as sc
 
 
+def make_var() -> sc.Variable:
```

Comment:
These functions can also be fixtures and request each other: https://docs.pytest.org/en/latest/how-to/fixtures.html#fixtures-can-request-other-fixtures This would avoid the need for calling them in the test body.

Reply:
I failed to see in the docs how I would parametrize `sliceable` by these then, since I cannot call them?

Reply:
Does not seem to be in the docs, but fiddling around with fixtures a bit gave this:
```.py
@pytest.fixture
def asd():
    return 'asd'


@pytest.fixture
def qwe():
    return 'qwe'


@pytest.fixture(params=['asd', 'qwe'])
def either(request):
    return request.getfixturevalue(request.param)
```
Judging by the docstring of `getfixturevalue`, this is how it is supposed to be used: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.FixtureRequest.getfixturevalue

Reply:
That seems hacky and complicated, no one can remember that. I think I'd prefer leaving it as it is.